shared/chat/container.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import Chat from './index'
+
+import {loadInbox} from '../actions/chat'
+
+class ChatContainer extends Component {
+  componentWillMount() {
+    this.props.loadInbox()
```

Comment:
lets's leave this for now. currently the inbox doens't get updates correctly so let's wait until thats actually working